src/inmanta/env.py

```diff
@@ -91,7 +91,7 @@ def init_env(self) -> bool:
 
             # --clear is required in python prior to 3.4 if the folder already exists
             try:
-                venv.create(path, clear=True, with_pip=True)
```

Comment:
The normal debug procedure of 'run pip in the venv' will no longer work, perhaps we have to put in place a fake pip

Reply:
fixed